docs/root/start/quick-start/admin.rst

```diff
@@ -0,0 +1,256 @@
+.. _start_quick_start_admin:
+
+Envoy admin interface
+=====================
+
+The optional admin interface provided by Envoy allows you to view configuration and statistics, change the
+behaviour of the server, and tap traffic according to specific filter rules.
+
+The admin interface can be configured for static and dynamic setups.
```

Comment:
hmm

i added this line because the next says about how useful it is to use this with dynamic config, and i feared it would be confusing. I think we do want to say something...

i reckon we can remove this line, and move the next down into the `config_dump` section

ill update...